lib/ts/BufferWriter.h

```diff
@@ -244,6 +244,22 @@ class FixedBufferWriter : public BufferWriter
 
   operator string_view() const { return view(); }
 
+  // When possible, provide access to buffer contents as a nul-terminated string.  The next non-const member function call
```

Comment:
I don't think this is useful. Much better to use `"%.*s` with `b.size(),buff` instead where `b` is the `BufferWriter` instance and `buff` is the buffer it's writing on. It would be reasonable to add a `char* data()` method that returned a pointer to the first byte in the writer's buffer, so you could do
```
printf("Buffer content - %.*s\n", b.size(), b.ptr());
```